# ----------------Image ans Mask params------------------
rho: -0.785398/2
w_max: 0.20
hori_strips: 10
sensorwidth_mm: 90.0
fov: 69.4*3.1415926536/180

ty: 1.075
tz: 1.275

fx: 612.7745361328125
fy: 612.6051635742188

# ----------------Controller Params------------------
lambda_x: 1.0 #Gain v_x
lambda_w: 1.1 #Gain w_x

controller_ID: 0
v: 0.3
